USER_GUIDE.md

```diff
@@ -105,3 +247,86 @@ appengine {
   }
 }
 ```
+
+---
+
+## App Engine Flexible
+The plugin will include App Engine standard features if you do **NOT** include an `appengine-web.xml`
```

Comment:
flexible

also similar to std: `The plugin will include App Engine flexible features` -> `The plugin will treat the project as targeted for App Engine flexible environment`?